app/models/manageiq/providers/hawkular/middleware_manager/event_catcher/runner.rb

```diff
@@ -1,6 +1,21 @@
 class ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcher::Runner <
   ManageIQ::Providers::BaseManager::EventCatcher::Runner
-  INCLUDE_EVENTS = {'Hawkular Deployment' => '.*'}.freeze
+
+  def initialize(cfg = {})
+    super(cfg)
+
+    # Accept events with an event category in the keyset and whose event message matches the (optional) pattern value
+    # - key = event category
+    # - value (optional) event message regex
+    @whitelist = {
+      'Hawkular Deployment' => nil
```

Comment:
I'm removing regex pattern stuff in the next PR.  I thought up front it would be a nice feature but so far it's unused.  So, please just ignore this issue.